code/WorkInProgress/TempEngine.dm

```diff
@@ -841,6 +874,40 @@ datum/pump_ui/circulator_ui
 
 		return efficiency_scale * 0.01
 
+	attackby(obj/item/W as obj, mob/user as mob)
+		// Weld > Crowbar > Rods > Weld
+		switch(semiconductor_state)
+			if(TEG_SEMI_STATE_PRESENT)
+				if (istool(W, TOOL_SCREWING))
+					actions.start(new /datum/action/bar/icon/teg_semiconductor_removal(src, W, 50), user)
```

Comment:
Why not use `5 SECONDS` instead of `50` here? (Same for the other actions.)

Reply:
Missed these as part of ZeWaka's same comment.  Good catch. 😊